helm/grid-intensity-exporter/templates/deployment.yaml

```diff
@@ -34,6 +34,17 @@ spec:
             configMapKeyRef:
               name: {{ .Release.Name }}
               key: gridIntensityLocation
+        - name: GRID_INTENSITY_NODE
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        {{- if .Values.gridIntensity.region }}
+        - name: GRID_INTENSITY_REGION
+          valueFrom:
+            configMapKeyRef:
+              name: {{ .Release.Name }}
+              key: gridIntensityRegion
```

Comment:
For k8s the region has to be set by the user. As there is no easy way to get the node label even if its set which is a pain.